src/plan_expand_hypertable.c

```diff
@@ -158,6 +221,88 @@ ts_plan_expand_hypertable_valid_hypertable(Hypertable *ht, Query *parse, Index r
 	return true;
 }
 
+/*  get chunk oids specified by explicit chunk exclusion function */
+static List *
+get_explicit_chunk_oids(CollectQualCtx *ctx, Hypertable *ht)
+{
+	List	   *chunk_oids = NIL;
+	Const	   *chunks_arg;
+	ArrayIterator chunk_id_iterator;
+	Datum		elem = (Datum) NULL;
+	bool		isnull;
+	Expr	   *expr;
+
+	Assert(ctx->chunk_exclusion_func->args->length == 2);
+	expr = lsecond(ctx->chunk_exclusion_func->args);
+	if (!IsA(expr, Const))
+		ereport(ERROR,
+				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+				 errmsg("second parameeter to chunk_in should contain only integer consts")));
```

Comment:
parameeter -> parameter

but maybe `argument` is actually the accurate terminology?